xcdat/regridder/xesmf.py

```diff
@@ -200,6 +200,6 @@ def horizontal(self, data_var: str, ds: xr.Dataset) -> xr.Dataset:
         # preserve non-spatial bounds
         output_ds = preserve_bounds(ds, self._output_grid, output_ds)
 
-        output_ds = output_ds.bounds.add_missing_bounds()
+        output_ds = output_ds.bounds.add_missing_bounds(axes=["X", "Y", "T"])
```

Comment:
```suggestion
        output_ds = output_ds.bounds.add_missing_bounds(axes=["X", "Y"])
```

Do we need to include `T` here? We don't include it by default in `open_dataset`. The bounds also don't appear to get dropped during horizontal regridding (so if they are in the original dataset, they are preserved during regridding).

Reply:
I included `T` here because the old `add_missing_bounds()` added bounds for X, Y, T, and Z (if these axes are present in the dataset). I included all of these axes (except `Z` because I forgot to here) to make sure `output_ds` would be the same.

Reply:
`add_missing_bounds()` originally added bounds for all axes including "T" and "Z".
To avoid regression, I decided to include "T" (and I forgot to include "Z") in the new `axes` argument.

I'm not sure if it is absolutely required to add "T" bounds during horizontal regridding. @jasonb5 any thoughts?

Reply:
I don't think `T` (or `Z`) should be added here. It would kind of be a surprise to not have time bounds on open and then have them added during regridding.

Reply:
Gotcha, makes sense to me. If I recall correctly, there was a discussion somewhere with Karl where somebody mentions that trying to add "Z" bounds using the generic midpoint algorithm can produce incorrect results.